api/Permissions.json

```diff
@@ -891,4 +927,4 @@
       }
     }
   }
-}
+}
```

Comment:
Oh wow, I honestly never knew this was a thing to be concerned about! I have removed the odd newline here and there because I thought they were unnecessary.

I've updated my config so VSCode adds newlines to the end of all files on save, and added the missing ones into this PR. Sorted.